experiments/yarn/src/main/scala/org/apache/gearpump/experiments/yarn/master/AmActor.scala

```diff
@@ -341,7 +341,13 @@ object YarnApplicationMaster extends App with ArgumentsParser {
         LOG.error("Caught exception", throwable)
         throwable.printStackTrace()
     }
+  }
 
+  private def fetchConfigOnHDFS(path: String) = {
```

Comment:
Please use HDFs distributed cache to distribute the conf.

1. Create a conf folder under HDFS://user/geapump/
2. upload the jars to that hdfs conf folder.
3. Add the conf folder to local Resource
Example:
```
+    Map(
+      "pack" -> newYarnAppResource(fs, new Path(s"$hdfsRoot/$version.tar.gz"),
+        LocalResourceType.ARCHIVE, LocalResourceVisibility.PUBLIC))
       conf" -> newYarnAppResource(fs, new Path(s"$hdfsRoot/conf"),
+        LocalResourceType.FILETYPE, LocalResourceVisibility.PUBLIC))
```
4. Then during runtime, the conf folder will be under container working directory.

Then you need to add conf into classpath of YarnAM, Master, and Worker.